kombu/asynchronous/aws/sqs/connection.py

```diff
@@ -25,6 +29,76 @@ def __init__(self, sqs_connection, debug=0, region=None, **kwargs):
             **kwargs
         )
 
+    def _create_query_request(self, operation, params, queue_url, method):
+        params = params.copy()
+        if operation:
+            params['Action'] = operation
+
+        # defaults for non-get
+        param_payload = {'data': params}
+        if method.lower() == 'get':
+            # query-based opts
+            param_payload = {'params': params}
+
+        return AWSRequest(method=method, url=queue_url, **param_payload)
+
+    def _create_json_request(self, operation, params, queue_url):
+        params = params.copy()
+        params['QueueUrl'] = queue_url
+
+        service_model = self.sqs_connection.meta.service_model
+        operation_model = service_model.operation_model(operation)
+
+        url = self.sqs_connection._endpoint.host
+
+        headers = {}
+        # Content-Type
+        json_version = operation_model.metadata['jsonVersion']
+        content_type = f'application/x-amz-json-{json_version}'
+        headers['Content-Type'] = content_type
+
+        # X-Amz-Target
+        target = '{}.{}'.format(
```

Comment:
can we try f string here? or there are some version restrictions?

Reply:
To be honest, I think this is cleaner, since the placeholders would be long, making it hard to read:

```
f'{operation_model.metadata['targetPrefix']}.{operation_model.name}'
```

especially since that it would make the line longer than 80 chars, meaning that we will need to split the line, or otherwise define variables to make it shorter.

Let me know if you feel strongly about it.